cicd/gitlab/st/statusTracking.py

```diff
@@ -65,23 +65,21 @@ def parse_result(listOfTasks, checkPublication=False):
             task['pubSummary'] = '%d/%d/%d' % (failedPublications, published_in_transfersdb, published_in_dbs)
 
             if total_jobs > 0 and ('finished', total_jobs) in task['jobsPerStatus'].items():
-                if task['status'] == 'COMPLETED':
-                    result = 'TestPassed'
-                else:
-                    needToResubmit = True
                 if checkPublication and 'nopublication' not in task['taskName']:
-                    # remove probe jobs (job id of X-Y kind) if any from count
-                    jobsToPublish = total_jobs
-                    for job in task['jobs'].keys():
-                        if '-' in job:
-                            jobsToPublish -= 1
+                    # Remove probe jobs (job id of X-Y kind) from count
+                    probe_jobs_count = sum(1 for job in task['jobs'].keys() if '-' in job)
+                    jobsToPublish = total_jobs - probe_jobs_count
                     if published_in_transfersdb == jobsToPublish and published_in_dbs == jobsToPublish:
                         result = 'TestPassed'
                     elif failedPublications:
                         #result = 'TestFailed'
                         needToResubmitPublication = True
                     else:
                         result = 'TestRunning'
+                if task['status'] == 'COMPLETED':
```

Comment:
I think that this simply discards whatever publication count was done before and says "passed" if status is COMPLETED, regardless of publication.
Or am I lost indentations ?